components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/service/notification/NotificationTemplateManager.java

```diff
@@ -34,5 +36,46 @@
      * @throws NotificationTemplateManagerException Error while getting the notification template
      */
     NotificationTemplate getNotificationTemplate(String notificationChannel, String templateType, String locale,
-            String tenantDomain) throws NotificationTemplateManagerException;
+                                                 String tenantDomain) throws NotificationTemplateManagerException;
+
+    /**
+     * Add the notification template to the registry.
```

Comment:
Please format the doc comment. (Try using IntelliJ Idea)

Reply:
Already added and formatted. Therefore, resolving the comment.